src/pages/candy-machine/guards/freeze-token-payment.md

```diff
@@ -116,7 +116,7 @@ Add this object into the guard section your config.json file:
 
 ```json
 "freezeTokenPayment" : {
-    "amount": number,
+    "amount": number number in basis points (e.g. 1000 for 1 Token that has 3 decimals),
```

Comment:
I think there's a double "number" now haha.

Reply:
(╯°□°)╯︵ ┻━┻

Thank you!